common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -252,6 +257,17 @@ def clear_existing_modulestores():
     _MIXED_MODULESTORE = None
 
 
+if XBlockConfiguration and disabled_xblocks:
```

Comment:
Previously disabled_xblock_types was only set on creation of the modulestore (above). Any configuration changes required a restart of the LMS.